packages/@aws-cdk/aws-lambda/lib/sns-dlq.ts

```diff
@@ -0,0 +1,19 @@
+import * as sns from '@aws-cdk/aws-sns';
+import { DLQDestinationConfig, IEventSourceDLQ } from "./dlq";
+
+/**
+ * An SNS dead letter queue destination configuration
+ */
+export class SnsDLQ implements IEventSourceDLQ {
```

Comment:
What do you think about putting the `SnsDLQ` and `SqsDLQ` classes in the lambda-event-sources package? 

Seems more appropriate given that it will be used only when configuring an event source.

Reply:
Had to compromise to avoid a circular dependency between aws-lambda and aws-lambda-event-sources so I ended up pulling it up into aws-lambda

Reply:
Not sure I follow where the circular dependency is coming from. 

I don't mean to move the `IEventSourceDlq` but only `SqsDlq` and `SnsDlq`.

This should be very similar to the modelling of `IEventSource` and the various implementations of it, no?

Reply:
Ah yeah that makes more sense!